src/pages/index.js

```diff
@@ -58,41 +148,66 @@ addCardButton.addEventListener("click", () => {
   newCardPopup.open();
 });
 
-// Initialize an instance
-newCardPopup.setEventListeners();
-
-// UserInfo popup
-// Comment to the reviewer - I changed here the logics to use
-// the PopupWithFormClass for Profile Data change
-// and removed the extension of the popup class to the UserInfo class
-
+// UserInfo Popup
 const userPopupForm = new PopupWithForm({
   popupSelector: "#profileChange",
   handleFormSubmit: (userInput) => {
-    const newUserInput = userInput;
-    userInfo.setUserInfo({
-      name: newUserInput.name,
-      description: newUserInput.description,
-    });
-    userPopupForm.close();
+    userPopupForm.renderLoading(true);
+    api
+      .editProfileData(userInput)
+      .then((data) => {
+        userInfo.setUserInfo({
+          name: data.name,
+          description: data.about,
+        });
+      })
+      .catch((err) => console.error(err))
+      .finally(() => userPopupForm.renderLoading(false));
   },
 });
 
+// PROFILE INFO CHANGES
+
+// User Info and Avatar Change
 const userInfo = new UserInfo({
   nameSelector: ".profile__title",
   descriptionSelector: ".profile__subtitle",
+  avatarSelector: ".profile__avatar",
+});
+
+//Profile Avatar Change
+const profilePicPopup = new ProfilePicPopup({
+  popupSelector: "#profilePicChange",
+  avatarSelector: ".profile__avatar",
+  handleFormSubmit: (inputValues) => {
+    profilePicPopup.renderLoading(true);
+    api
+      .avatarUpdate(inputValues)
+      .then((data) => {
+        profilePicPopup.changeProfilePic(data);
```

Comment:
* `UserInfo` handles avatar changing with `setNewAvatar`. You don't need to do that anywhere else